### PR TITLE
PHI Check

### PR DESCRIPTION
**Pull request by cameronmarlow**
* Add a list of PHI mailboxes to the script
* Check incoming posts to see if they are in one of these mailboxes, and if they do, post something innocuous.

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def lambda_handler(event, context):
         os.environ['FB_GROUP_ID'],
     )
 
+    sensitive_mailboxes = json.loads(os.environ['SENSITIVE_MAILBOXES'])
```
Comment (slycoder) [inline handler.py]: For safety, maybe check if SENSITIVE_MAILBOXES is falsey before trying to parse --- if it's not set we can just set sensitive_mailboxes to an empty dict.

Comment (cameronmarlow) [reply handler.py]: nice catch

Comment (cameronmarlow) [reply handler.py]: this ok?

```python
sensitive_mailboxes = json.loads(
        os.environ.get(['SENSITIVE_MAILBOXES'], "[]")
    )
```

Comment (slycoder) [reply handler.py]: You don't need the brackets around 'SENSITIVE_MAILBOXES'

**Pull request merged**